_posts/2015-11-15-sinatra-app.markdown

```diff
@@ -11,79 +11,166 @@ permalink: sinatra-app-guide
 ## What is Sinatra?
 <img src="/images/sinatrawiki.png">
 
-
 # Getting Started
 
-## *1.*Creating the application with Cloud9
+## *1.* Creating the application with Cloud9
 
 Follow the steps outlined in the Ruby Girls Cloud9 Sinatra Setup Guide [Cloud9 Setup](/cloud9-sinatra-guide)
 
-## *2.*Locate the terminal window in Cloud9
+## *2.* Write a description for your application
+
+Cloud9 automatically creates a file called `REAMDME.md`. The `.md` extension means that this is a MarkDown file.
+A readme file is useful for documenting what your application does, and how a developer can setup their environment to develop or run your application.
+There is no clearly defined Markdown standard, however, GitHub has their own [flavor](https://help.github.com/articles/github-flavored-markdown/)(sic).
+
+- [ ] Double click the `README.md` in the Cloud9 file explorer to open it.
+- [ ] Replace the contents of the file with your description.
+- [ ] Save the file.
+
+eg.
+```
+# My awesome Sinatra app!
+This app is like the RailsGirls Ideas app, but using Sinatra instead of Rails!
+```
+
+We will add more documentation as the project evolves.
+
+## *3.* Git
+
+While you are developing your app, you should be committing your changes and pushing them to a remote repository, such as GitHub.
+
+We'll start by initialising git within our app and adding the current files.
+
+- [ ] Run the git initialiser
+
+{% highlight sh %}
+git init
+{% endhighlight %}
+
+- [ ] Commit the app to git
+
+{% highlight sh %}
+git add .
+git commit -m "Initial commit"
+{% endhighlight %}
 
-There are three gems we will need to install before we can create a Sinatra project.
+You can modify the `Initial commit` message to anything you like.
+
+The next step is to create a remote repository on GitHub and add it to your app.
+
+- [ ] Log onto GitHub and create a new repository
+
+<img src="/images/github_new_repo.png">
+
+- [ ] Give your repo a title and a description
+
+<img src="/images/github_create_repo.png">
+
+- [ ] Follow the instructions on GitHub under the heading `…or push an existing repository from the command line`
+
+They will be similar to below. DO NOT copy the lines below. You must push to your own newly created repo
+
+{% highlight sh %}
+git remote add origin https://github.com/YourName/project_name.git
+git push -u origin master
+{% endhighlight %}
 
-To install this gem, enter the following commands in the terminal window:
```

Comment:
Wasn't sure why the sinatra gem was being installed globally, so i removed this part